tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/bulkhead/BulkheadFutureTest.java

```diff
@@ -93,11 +93,11 @@ public void testBulkheadMethodAsynchFutureDoneAfterGet() {
         Assert.assertFalse(result.isDone(), "Future reporting Done when not");
         try {
             String r;
-            Assert.assertTrue("GET.".equals(r = result.get()), r);
-            Assert.assertTrue("GET.GET_TO.".equals(r = result.get(1, TimeUnit.SECONDS)), r);
+            Assert.assertTrue("RESULT".equals(r = result.get()), r);
+            Assert.assertTrue("RESULT".equals(r = result.get(1, TimeUnit.SECONDS)), r);
```

Comment:
I think this (and perhaps other parts of this PR?) will conflict with #469. I'm fine with letting #469 in and then rebasing.